contacts/buttons.helper.js

```diff
@@ -0,0 +1,125 @@
+import {
+  MESSAGES,
+  DOCUMENT_ELEMENTS,
+  ZERO
+} from './constants';
+import ContactsService from './contacts.service';
+import { // eslint-disable-line
+  reRenderContactsListHTML,
+  resetDeleteAllButtonValueAndState,
+  clearFavouritesListHTML,
+  geInput,
+  setButtonDisabled,
+  setButtonEnabled,
+  clearFormInputs,
+  addedSuccessfullyPerson,
+  reRenderFavouritesListHTML
+} from './html-helper';
+import {
+  getPersonIdByHTMLElementId,
+  generateUniqueID,
+  isEmpty,
+  createPerson
+} from './helper';
+import {
+  validateName,
+  validateImageUrl
+} from './validations';
+
+
+const isCheckboxChecked = checkBoxId => document.getElementById(checkBoxId).checked;
+
+const getSelectedContactsIDs = () => ContactsService.internalArrayContacts
```

Comment:
This method should be implemented inside the ContactsService, since you are iterating trough contacts array. Also there are too many iterations. You can optimize this to use a single iteration.